src/clj/witan/send/send.clj

```diff
@@ -622,5 +628,8 @@
         (ch/ribbon-plot leaver-rates-CI "Leaver" years n-colours)
         (ch/ribbon-plot mover-rates-CI "Mover" years n-colours)
         (ch/population-line-plot transitions-data (map :total-in-send send-output))
-        (ch/send-cost-plot (map :total-cost send-output) years))))
+        (ch/send-cost-plot (map :total-cost send-output) years)
+        (u/write-log)
```

Comment:
I think if `write-log` is here, it will only output if `= output? true` in the run-model fn?

Reply:
Ah, yeah, good point. Do we want that, or should we output the summary in both cases? I can see arguments both ways.

Reply:
Yeah, good call... I think as we're actually returning less to screen, I think we'd *always* want the log taken. It's just stored in target isn't it? So if it wasn't important it'll just be written over next time and I am assuming it doesn't really slow anything down when this is output?